osler/workup/models.py

```diff
@@ -300,13 +228,13 @@ def written_date(self):
         '''
         Returns the date (not datetime) this workup was written on.
         '''
-        return self.clinic_day.clinic_date
+        return self.encounter.clinic_day
 
     def get_absolute_url(self):
         return reverse('workup', args=[str(self.id)])
 
     def __str__(self):
-        return self.patient.name() + " on " + str(self.clinic_day.clinic_date)
+        return self.patient.name() + " on " + str(self.encounter.clinic_day)
```

Comment:
This was displaying strangely on patient detail for me. It was showing something like "January 1, 2020, midnight"

Reply:
urhhhhhhhh yeah I think clinic_day may be saving a date and time instead of just the date

Reply:
I've hopefully fixed this by using `clinic_day.strftime` instead of just calling it's string method.

Reply:
I was wrong about which lines of code were causing this. It's because self.written_datetime() is being called in patient detail, but the fake workups in my database didn't have a time, so they defaulted to midnight. The code is right